duration.go

```diff
@@ -1,3 +1,8 @@
+// SPDX-License-Identifier: BSD-2-Clause
+// Provenance-includes-location: https://github.com/crewjam/saml/blob/a32b643a25a46182499b1278293e265150056d89/duration.go
+// Provenance-includes-license: BSD-2-Clause
+// Provenance-includes-copyright: Ross Kinder
```

Comment:
I would also prefer to have 2015-202X (depending on when X was) in those lines

Reply:
Done!